src/main/scala/org/ergoplatform/network/ModeFeature.scala

```diff
@@ -0,0 +1,65 @@
+package org.ergoplatform.network
+
+import org.ergoplatform.nodeView.state.StateType
+import org.ergoplatform.settings.{NodeConfigurationSettings, PeerFeatureIds}
+import scorex.core.network.PeerFeature
+import scorex.core.network.PeerFeature.Id
+import scorex.core.serialization.ScorexSerializer
+import scorex.util.serialization.{Reader, Writer}
+
+
```

Comment:
Please use correct formatting - one line instead of two. Same in few places below.